src/SourceLine.cpp

```diff
@@ -3,11 +3,11 @@
 #include "SourceFile.h"
 
 #include <algorithm>

```

Comment:
Interesting, so <iterator> is included by <algorithm> by both gcc and clang. I think you can remove the #ifdef as this header really should be included on all platforms.